public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -186,7 +192,7 @@ export const ServiceAccountPageUnconnected = ({
             />
           )}
           <div className={styles.tokensListHeader}>
-            <h4>Tokens</h4>
+            <h3>Tokens</h3>
```

Comment:
note: this is done to make `Tokens` and `Information` headers the same size as `Permissions` header.